qemu/tests/cgroup.py

```diff
@@ -2059,6 +2061,51 @@ def memory_move():
         else:
             return (out)
 
+    def cfs_bandwidth():
+        """
+        CFS bandwidth limit
+        """
+        def stress_thread():
+            session.cmd("stress -c 1 -t 100", 120)
+
+        logging.info("Setup test")
+        modules = CgroupModules()
+        if (modules.init(['cpu']) != 1):
+            raise exceptions.TestFail("Can't mount cpu cgroup modules")
+        cgroup = Cgroup('cpu', '')
+        cgroup.initialize(modules)
+
+        # Create VM with smp=1
+        params['smp'] = 1
+        params["vcpu_sockets"] = 1
+        vm_name = params["main_vm"]
+        params["start_vm"] = "yes"
+        env_process.preprocess_vm(test, params, env, vm_name)
+        vm = env.get_vm(params["main_vm"])
+        login_timeout = int(params.get("login_timeout", 360))
+        session = vm.wait_for_login(timeout=login_timeout)
+
+        cgroup.mk_cgroup()
+        cgroup.set_property("cpu.cfs_period_us", 100000, 0)
+        cgroup.set_property("cpu.cfs_quota_us", 50000, 0)
+        assign_vm_into_cgroup(vm, cgroup, 0)
+
+        logging.info("Test")
+        stress_t = threading.Thread(target=stress_thread)
+        stress_t.start()
+        count = 0
+        while count < 6:
+            time.sleep(10)
+            o = process.system_output("top -b -p %s -n 3 | tail -1" % vm.get_pid(),
```

Comment:
Although it is number actually needed, so you may not have problem on Python 3, it is better decode() the `bytes` output to `str`.